server/main.py

```diff
@@ -57,18 +58,17 @@ def api_claim_applicant(recruiter_id, applicant_id):
         Bad Request (400): If applicant_id is not an unclaimed applicant
     """
     # TODO: be sure to check that the applicant is in fact an unclaimed applicant
-    return jsonify(recruiter_claim_applicant(recruiter_id, applicant_id))
+    return jsonify(recruiter_claim_applicant(current_user.get_id(), applicant_id))
 
 
 @app.route(
-    '/api/recruiter/<int:recruiter_id>/<int:applicant_id>/release', methods=['GET'])
+    '/api/recruits/release/<int:applicant_id>', methods=['GET'])
 @login_required
-def api_release_applicant(recruiter_id, applicant_id):
```

Comment:
Similarly, should it be possible for a recruiter to claim applicants and refuse to release them, or should there be a way for a senior recruiter or admin to release applicants?

Unlike in claiming applicants I can actually keep the `release(applicant_id)` call signature while still allowing senior recruiters/admins to do this, but I need to know if that's desired.

Reply:
yes I think that's good. In case a recruiter is unavailable / leaves corp

Reply:
I started making this change, then realised the recruiter is the current user, so this will work as long as the ensure_has_access allows it, which it will for admins and snr recruiters, so I think that's what we need

Reply:
Yes, we just need to make sure the permissions don't require the logged-in user be the claiming recruiter.

Reply:
indeed. They need to be (and I think are) the claiming recruiter, or snr, or admin.

A good one for unit tests